### DISCUSSION TITLE
Cannot move custom segment marker

### DISCUSSION THREAD

**Discussion by SeanBannister** [bbc/peaks.js#526] [Q&A]
I've just implemented Custom Segment Markers by copying the code [from the demo](https://github.com/bbc/peaks.js/tree/master/demo/custom-markers). In the demo dragging one of the segment markers allows you to edit the segments position. However in my code dragging the segment marker doesn't do anything. I was wondering if someone could point me in the direction of what I'm doing wrong as I'm a little lost. Here's a stripped [down demo](https://codepen.io/SeanBannister/pen/rNRRmoo) of this issue.

**Comment by chrisn** [comment]
Please take a look at the [FAQ](https://github.com/bbc/peaks.js/blob/master/doc/faq.md).

**Comment by SeanBannister** [comment]
Brilliant, thank you!!
